npm-shrinkwrap.json

```diff
@@ -4233,6 +4238,11 @@
               "from": "isarray@>=1.0.0 <1.1.0",
               "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz"
             },
+            "isstream": {
```

Comment:
This dependency is declared on the root of the shrinkwrap file, however it looks like npm 3.10.10 insists that it should be duplicated inside the nested dependency that also requires it.